Makefile

```diff
@@ -78,6 +78,7 @@ else
 	else
 		ESPPORT = $(COMPORT)
 	endif
+	export PATH := $(PATH):$(TOP_DIR)/tools/toolchains/esp8266/bin/
```

Comment:
Note that this is exported only to shells that `make` creates to run commands; it does not escape and affect the shell that invoked `make`.